### PR TITLE
Add timeline

### PR DESCRIPTION
**Pull request by oskarth**
A lot of overlap with vac.comm / Dagger

It isn't clear to me exactly to what extent we want this to be the case, ideally the efforts are integrated (but "modified" when that domain requires it)

### CODE REVIEW

**Comment by oskarth** [comment]
@dryajov wdyt? (can't add review until you accepted invite)

File: vac-comm.md [hunk 1]
```diff
@@ -57,9 +57,58 @@ These are rough notes of things we currently know and don't know. They should be
...
+**Outstanding:**
+
+1. Get consensus on strategy, direction for most immediate users (Vac meetup, Dagger and Status Core)
+
+2. Create a detailed timeline for implementing the project as a deliverable
+
+**Overlap / in conjunction with Dagger priorities:**
+
+1. Research existing projects such as:
+        Swarm
```
Comment (decanus) [inline vac-comm.md]:
```suggestion
 - Swarm
```

File: vac-comm.md [hunk 2]
```diff
@@ -57,9 +57,58 @@ These are rough notes of things we currently know and don't know. They should be
...
+        IPFS/Filecoin
```
Comment (decanus) [inline vac-comm.md]:
```suggestion
 - IPFS/Filecoin
```

File: vac-comm.md [hunk 3]
```diff
@@ -57,9 +57,58 @@ These are rough notes of things we currently know and don't know. They should be
...
+        BitTorrent
```
Comment (decanus) [inline vac-comm.md]:
```suggestion
 - BitTorrent
```

**Comment by dryajov** [comment]
Looks good as a starting point. 

I've done some initial thinking around incentivization (very preliminary) - https://hackmd.io/iv6D-kO6QpWxcBW0-SXqWA.

Here are my initial thoughts around message incentivization:

> # Message Incentivization
>It turns out that incentivizing most topic based gossip mechanisms is unnecessary due to the the presence of interest group dynamics. In other words, if a node chooses to participate in a particular topic, it is because the peer is interested in the topic, so in effect no incentivization is required and it works similarly to how file sharing already works with popular content. In the same realm, incentivizing a conversation between two peers that are a priory interested in conversing is moot. This applies to messaging for example, if two peers are engaged in exchanging messages that both care about, they will default to skipping payments.

**Pull request merged**